Keas.Mvc/Handlers/VerifyRoleAccessHandler.cs

```diff
@@ -34,13 +35,11 @@ protected override async Task HandleRequirementAsync(AuthorizationHandlerContext
         {
             // get team name from url
             var team = "";
-            if (context.Resource is AuthorizationFilterContext mvcContext)
+            if (context.Resource is Endpoint)
             {
-                if (mvcContext.RouteData.Values["teamName"] != null)
-                {
-                    team = mvcContext.RouteData.Values["teamName"].ToString();
-                }
+                team = _httpContext.HttpContext.Request.RouteValues["teamName"].ToString();
```

Comment:
@sprucely so at this point are we guaranteed that teamName is never null and we don't have to check anymore?

Reply:
Never saw a null during debugging, but lots of empty strings. I'll add a null coalesce just to be safe...